app/models/student.rb

```diff
@@ -113,6 +113,32 @@ def eligible_for_trip(trip)
       has_consented_stim && has_signed_safety_test
   end
 
+  def has_forms_for_year
+    #Checks to see if a student has valid forms for this year.
+    #Returns the personal email of the student and array of missing forms.
+    year = Date.current.year
```

Comment:
This is some duplicate logic from elsewhere in this file. Seems like a good candidate to factor out.